deflect/SizeHints.h

```diff
@@ -73,6 +73,10 @@ struct SizeHints
     //@}
 };
 
+#if _MSC_VER
+#  define noexcept
```

Comment:
Arff, partial C++11 support :-(
Should we add a version check to that so that it eventually gets deprecated, like:
if defined(_MSC_VER) && _MSC_VER < 1800 ?
http://stackoverflow.com/questions/18387640/how-to-deal-with-noexcept-in-visual-studio